src/Aquifer.Common/Messages/Publishers/ResourceContentRequestTrackingMessagePublisher.cs

```diff
@@ -71,13 +72,12 @@ public async Task PublishTrackResourceContentRequestMessageAsync(
         }
 
         var sourceHeader = httpContext.Request.Headers["bn-source"].ToString();
-        httpContext.Items.TryGetValue(Constants.HttpContextItemCachedApiKey, out var maybeCachedApiKey);
 
         var message = new TrackResourceContentRequestMessage
         {
             Source = string.IsNullOrEmpty(sourceHeader) ? source : sourceHeader,
             IpAddress = GetClientIp(httpContext),
-            SubscriptionName = maybeCachedApiKey is CachedApiKey cachedApiKey ? cachedApiKey.Organization : null,
+            SubscriptionName = _cachingApiKeyService.CurrentApiKey.Organization,
```

Comment:
I'm assuming that the API Key can never be null here.  Any access to resource data (via an endpoint) requires an API Key.  Jobs doesn't publish these.